packages/base-styles/_variables.scss

```diff
@@ -93,11 +93,11 @@ $widget-area-width: 700px;
  */
 
 $border-width: 1px;
-$block-controls-height: 36px;
-$icon-button-size: 36px;
+$block-controls-height: 36px; // @todo: This should be deprecated in favor of 48px block control heights.
+$icon-button-size: 36px; // @todo: This should be deprecated in favor of 48px block control heights.
```

Comment:
Can you elaborate on what you mean here?

Reply:
I mean I don't really understand why this is deprecated. The button size is still 36px across the UI (aside toolbars)

Reply:
Oh, good point. I suppose I'll remove those Todos for now. I think there is some variable cleanup todo here so that there isn't any confusion — "icon button" could refer to formatting buttons which should be 48x48, also on mobile, and perhaps even in the toolbar, wheras yes, the traditional blue primary and secondary buttons are 36 in height. 

I'm speaking specifically about the clickable hit area also.